torch/_torch_docs.py

```diff
@@ -5590,6 +5590,28 @@ def merge_dicts(*dicts):
     tensor([ 1., -1.,  0.,  1.])
 """.format(**common_args))
 
+add_docstr(torch.signbit,
+           r"""
+signbit(input, out=None) -> Tensor
+
+Returns a new tensor with the signbits of the elements of :attr:`input`.
```

Comment:
We have a new template for functions like this!

"Tests if each element of :attr:`input` has its sign bit set (is less than zero) or not."

Reply:
Addressed, thanks for advice!